recipes/evalhyd-cpp/meta.yaml

```diff
@@ -0,0 +1,56 @@
+# name for conda package (i.e. `conda install -c conda-forge <project_name>`)
+{% set project_name = "evalhyd-cpp" %}
+# name for C++ library
+{% set library_name = "evalhyd" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "eacfa5ad3bad30cb3af7b5584a0c1183a89de109fd252d0d4cfca47db891fdae" %}
+
+package:
+  name: {{ project_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.irstea.fr/HYCAR-Hydro/evalhyd/{{ project_name }}/-/archive/v{{ version }}/{{ project_name }}-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Probably should have a run-export
```
  run_exports:
    - {{ pin_subpackage("libevalhyd", max_pin="x.x") }}
```
`x.x.x`  or `x.x` depending on whether you expect the ABI to break in patch releases or not

Reply:
added with `x.x` only, patches not expected to break the ABI